eZ/Publish/SPI/Tests/FieldType/MediaIntegrationTest.php

```diff
@@ -80,8 +80,9 @@ public function getCustomHandler()
                     'LegacyStorage' => new FieldType\Media\MediaStorage\Gateway\LegacyStorage(),
                 ),
                 $this->ioService = self::$container->get( "ezpublish.fieldType.ezbinaryfile.io_service" ),
-                new FieldType\BinaryBase\PathGenerator\LegacyPathGenerator(),
-                new FileInfo()
+                $legacyPathGenerator = new FieldType\BinaryBase\PathGenerator\LegacyPathGenerator(),
+                new FileInfo(),
+                $legacyPathGenerator
```

Comment:
AFAICS there is not fifth parameter here, so should be removable, together with assigned variable.